actions/lock-namespace/entrypoint.sh

```diff
@@ -15,6 +15,7 @@ while true; do
             echo "Namespace ${NS} not in use -- Locking"
             kubectl create secret generic -n "${NS}" "${NS}-inuse" --from-literal=inuse="${NS}"
             echo "NAMESPACEINUSE=${NS}" >> "$GITHUB_ENV"
+            echo "::set-output name=NAMESPACEINUSE::${NS}"
```

Comment:
`echo "NAMESPACEINUSE=${NS}" >> "$GITHUB_ENV"` will export the variable to the environment, so I don't think the extra line is necessary.

Reply:
The action failed when I removed this line 😞:
```
+ echo 'NAMESPACEINUSE: '
...
NAMESPACEINUSE: 
Error: "helm uninstall" requires at least 1 argument
```

Here's the test run: https://github.com/DataBiosphere/jade-data-repo/runs/2152307554?check_suite_focus=true

Reply:
Interesting, it seems like it was wiped in a previous step: https://github.com/DataBiosphere/jade-data-repo/runs/2152307554?check_suite_focus=true#step:15:12

Reply:
I see the problem. This line causes the issue: https://github.com/DataBiosphere/jade-data-repo/actions/runs/669318446/workflow#L107 . If you remove the `env` section then it should work 👍

Reply:
Oh I see! Thanks for looking into this!